t3f/ops.py

```diff
@@ -802,20 +802,26 @@ def frobenius_norm_squared(tt, differentiable=False):
     sum of squares of all elements in `tt`
```

Comment:
Add to the comments that the input maybe a batch and the output in this case will be a tensor with norms for each tt